tests/models/x_clip/test_modeling_x_clip.py

```diff
@@ -547,6 +547,14 @@ class XCLIPModelTest(ModelTesterMixin, PipelineTesterMixin, unittest.TestCase):
 
     def setUp(self):
         self.model_tester = XCLIPModelTester(self)
+        common_properties = ["projection_dim", "prompt_layers", "prompt_num_attention_heads"]
+        self.config_tester = ConfigTester(
+            self, config_class=XCLIPConfig, has_text_modality=False, common_properties=common_properties
+        )
+
+    def test_config(self):
+        self.config_tester.run_common_tests()
+        self.config_tester.create_and_test_config_from_and_save_pretrained_composite()
```

Comment:
do we have to add this? Can't we add this as part of the run_common? (I mean this should always be on by default without us having to write anything given the CONFIG_MAPPING and model types that are available to us!

Reply:
Could be but the issues is in other tests that fail in case of composite models due to various reasons. We could maybe skip those tests if the model has non-empty `sub_configs`, lemme see